docs/_pages/installation/roborock.md

```diff
@@ -212,7 +212,10 @@ Connect the marked TPA17 to GND using your jumper cable or paperclip. You can us
 The marked SH1 for GND should be close enough to enable you to do the procedure without the help of another person.
 There might be some conformal coating on TPA17. You can use your fingernails to carefully scrape that off.
 
-Press the power button for 3 seconds and keep the TPA17 connected to GND for 5 more seconds after that.
+For the Roborock S7, the marked area can be used as GND as well:  
+![S7 Mainboard testpoint for FEL root](./img/s7_board_with_testpoint_for_fel.jpg)
+
+Press the power button for 3 seconds and keep the TPA17 connected to GND for 5 more seconds after that. Make sure to properly press the power button - if pressed correctly, the LEDs should light up after some seconds.
```

Comment:
Is this really an issue so common, that it warrants mentioning in this guide though?

Similar to what I said above, more information unfortunately doesn't equal better documentation. Attention spans are limited and thus people will tune out if there's just too much text.
It's also not feasible to account for every possible mistake one could make in the docs.

I can imagine the frustrating experience that led to this change but in the end you did manage to figure that out by yourself so I'd say that anyone else who might be affected also will do so.